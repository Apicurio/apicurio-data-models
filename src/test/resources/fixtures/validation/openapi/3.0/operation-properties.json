{
    "openapi": "3.0.2",
    "info": {
        "title": "Missing Property Test",
        "version": "1.0.0"
    },
    "paths": {
        "/test": {
            "get": {
                "tags": [
                    "foo"
                ],
                "responses": {
                    "200": {
                        "description": "200 OK"
                    }
                },
                "operationId": "getTest",
                "description": "Gets the test."
            },
            "delete": {
                "responses": {
                    "204": {
                        "description": "204 No Content"
                    }
                },
                "operationId": "deleteTest",
                "summary": "Delete Test"
            }
        }
    }
}