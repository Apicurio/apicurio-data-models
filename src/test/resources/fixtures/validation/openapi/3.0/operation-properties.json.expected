[OP-014] |high| {/paths[/test]/get->summary} :: Operation is missing a summary.
[OP-015] |high| {/paths[/test]/delete->description} :: Operation is missing a description.
[OP-016] |high| {/paths[/test]/delete->tags} :: Operation must have at least one tag.